tests/src/lv_test_conf_full.h

```diff
@@ -101,6 +101,9 @@
 
 #define LV_CACHE_DEF_SIZE       (10 * 1024 * 1024)
 
+#define LV_USE_LINUX_DRM    1
```

Comment:
Is it possible to make the test can run on mac?
I  have made a try in https://github.com/lvgl/lvgl/pull/5672. Hope it's the way.

Reply:
Good call. I tried to mimic your approach here.